src/components/search-form/search-filters.js

```diff
@@ -39,7 +44,64 @@ const SearchFilters = ({
   onUpdate,
   closedByDefault,
   disabled,
+  hasAccordion = true,
 }) => {
+  const labelRef = useRef(null);
+
+  const handleClearFilter = (name) => {
+    onUpdate({
+      ...activeFilters,
+      [name]: undefined,
+    });
+  };
+
+  const handleClearButtonClick = (labelId, name) => {
+    handleClearFilter(name);
+
+    // focus on the default option
+    setTimeout(() => {
+      document.querySelector(`[aria-labelledby="${labelId}"] input[tabindex="0"]`)?.focus();
```

Comment:
Can we use `autoFocus={isChecked}` on `<RadioButton>` instead of this call?

Reply:
We need to move the focus after hitting the clear icon, this will happen on page update, and `autoFocus` is only for page mounting.